src/kio/static/protocol.py

```diff
@@ -1,12 +1,45 @@
+from __future__ import annotations
+
 from typing import ClassVar
 from typing import Protocol
+from typing import TypeAlias
```

Comment:
Nit, but aren't we missing `isort`? 🤔

Reply:
Separate PR for this, I presume it would change few files and should be chore of its own to do so.

Reply:
For sure, adding it doesn't belong here, just stood out :)

Reply:
Already configured: https://github.com/Aiven-Open/kio/blob/2f9d9cc4c34b42a8481ebf89a79e9b381cb7dbae/pyproject.toml#L137